prow/crier/reporters/gerrit/reporter.go

```diff
@@ -309,19 +316,27 @@ func ParseReport(message string) *JobReport {
 	if !isReport {
 		return nil
 	}
-	report := &JobReport{}
-	report.header = contents[start]
-	for i := start; i < len(contents); i++ {
-		j := deserializeJob(contents[i])
-		if j != nil {
-			report.Jobs = append(report.Jobs, j)
+	var report JobReport
+	report.Header = contents[start] + "\n"
+	for i := start + 1; i < len(contents); i++ {
+		if contents[i] == "" {
+			continue
 		}
+		var j Job
+		if err := deserialize(contents[i], &j); err != nil {
+			logrus.Warnf("Could not deserialize %s to a job: %v", contents[i], err)
+		}
+		report.Total++
+		if j.State == v1.SuccessState {
+			report.Success++
+		}
+		report.Jobs = append(report.Jobs, j)
 	}
-	report.message = strings.TrimPrefix(message, report.header)
-	return report
+	report.Message = strings.TrimPrefix(message, report.Header+"\n")
```

Comment:
`report.Header` already has the extra `\n` on line 320, so this one is extraneous? (also might fail to trim properly) 
or am I missing something?